Update LoqedAPI to handle invalid transitions better

mikewoudenberg opened the pull request:
## Proposed change
Updated the LoqedAPI companion library which handles invalid transitions better

https://github.com/cpolhout/loqedAPI/compare/v2.1.7...v2.1.8

## Type of change

- [x] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New integration (thank you!)
- [ ] New feature (which adds functionality to an existing integration)
- [ ] Deprecation (breaking change to happen in the future)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes issue: fixes #99384


## Checklist
<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your code.
-->

- [x] The code change is tested and works locally.
- [x] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [x] I have followed the [development checklist][dev-checklist]
- [x] I have followed the [perfect PR recommendations][perfect-pr]
- [x] The code has been formatted using Black (`black --fast homeassistant tests`)
- [] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

If the code communicates with devices, web services, or third-party tools:

- [x] The [manifest file][manifest-docs] has all fields filled out correctly.  
      Updated and included derived files by running: `python3 -m script.hassfest`.
- [x] New or updated dependencies have been added to `requirements_all.txt`.  
      Updated by running `python3 -m script.gen_requirements_all`.
- [x] For the updated dependencies - a link to the changelog, or at minimum a diff between library versions is added to the PR description.
- [ ] Untested files have been added to `.coveragerc`.

<!--
  This project is very active and we have a high turnover of pull requests.

  Unfortunately, the number of incoming pull requests is higher than what our
  reviewers can review and merge so there is a long backlog of pull requests
  waiting for review. You can help here!
  
  By reviewing another pull request, you will help raise the code quality of
  that pull request and the final review will be faster. This way the general
  pace of pull request reviews will go up and your wait time will go down.
  
  When picking a pull request to review, try to choose one that hasn't yet
  been reviewed.

  Thanks for helping out!
-->

To help with the load of incoming pull requests:

- [ ] I have reviewed two other [open pull requests][prs] in this repository.

[prs]: https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-desc+review%3Anone+-status%3Afailure

<!--
  Thank you for contributing <3

  Below, some useful links you could explore:
-->
[dev-checklist]: https://developers.home-assistant.io/docs/development_checklist/
[manifest-docs]: https://developers.home-assistant.io/docs/creating_integration_manifest/
[quality-scale]: https://developers.home-assistant.io/docs/integration_quality_scale_index/
[docs-repository]: https://github.com/home-assistant/home-assistant.io
[perfect-pr]: https://developers.home-assistant.io/docs/review-process/#creating-the-perfect-pr

mikewoudenberg committed:
Update LoqedAPI to handle invalid transitions better

frenck reviewed (commented):
I don't see any releases on GitHub. Would be nice if the upstream package did that, so we can compare changes and see diffs.

../Frenck

mikewoudenberg commented:
> I don't see any releases on GitHub. Would be nice if the upstream package did that, so we can compare changes and see diffs.
> 
> ../Frenck

I added releases and updated the PR

frenck commented:
> I added releases and updated the PR


![CleanShot 2023-10-06 at 15 25 29](https://github.com/home-assistant/core/assets/195327/b8ea90c4-f97d-49a5-a2a0-d8e23df7f5df)


They don't show up. Something went wrong?

../Frenck

mikewoudenberg commented:
> > I added releases and updated the PR
> 
> ![CleanShot 2023-10-06 at 15 25 29](https://user-images.githubusercontent.com/195327/273221540-b8ea90c4-f97d-49a5-a2a0-d8e23df7f5df.png)
> 
> They don't show up. Something went wrong?
> 
> ../Frenck

I fiddled around with the release thing a bit more and I see them:
<img width="1203" alt="Screenshot 2023-10-06 at 15 32 41" src="https://github.com/home-assistant/core/assets/5583336/f7ada343-2773-4d59-a3a3-7247faa50b4d">

frenck commented:
Aah looks like a caching thing @ GitHub or something. If I enter the URLs they are indeed there.

Ok, thanks! 👍

frenck reviewed (approved):
Thanks, @mikewoudenberg 👍 

../Frenck
